base/mac/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+if (is_mac) {
```

Comment:
`assert(is_apple)` and you can drop the other checks

Reply:
Fixed.